src/core/Table/cells/DefaultCell.tsx

```diff
@@ -4,10 +4,12 @@
  *--------------------------------------------------------------------------------------------*/
 import React from 'react';
 import { CellRendererProps } from 'react-table';
+import cx from 'classnames';
 
-export type DefaultCellProps<
-  T extends Record<string, unknown>
-> = CellRendererProps<T> & React.ComponentPropsWithoutRef<'div'>;
+export type DefaultCellProps<T extends Record<string, unknown>> = {
+  disabled?: (rowData: T) => boolean;
+} & CellRendererProps<T> &
```

Comment:
If `isRowDisabled` is passed, it is being used in selection action handler, which makes active checkbox not selectable. So how can I reuse `isRowDisabled` if I can't pass it to the Table?

Reply:
> If isRowDisabled is passed, it is being used in selection action handler, which makes active checkbox not selectable.

Can this behavior be changed if user has custom columns? Would be nice to utilize the `isRowDisabled` prop instead of just saying "this prop should not be used if you have custom columns".

Reply:
Taking state from the row now.